libs/shared/left-side-menu/src/lib/shared-left-side-menu.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+
+import './shared-left-side-menu.scss';
+
+import {
+  SwipeableDrawer,
+  List,
+  ListItem,
+  Typography,
+  ListItemText
+} from '@material-ui/core';
+
+export interface SharedLeftSideMenuProps {
+  menuItemsArray: Array<Type>
+}
+
+interface Type {
+  name: string;
+  path?: string
+}
+
+export const SharedLeftSideMenu = (props: SharedLeftSideMenuProps) => {
+         const [state, setState] = React.useState({
+           menuOpen: true,
+         });
+
+         const menuItems = props.menuItemsArray;
+
+         const toggleDrawer = (open: boolean) => (
+           event: React.KeyboardEvent | React.MouseEvent
```

Comment:
don't use close drawer on menu click, i recommend use extra button on top for opening and closing drawer. ex: https://material-ui.com/components/drawers/#persistent-drawer